docgenerator/src/main/scala/codacy/codenarc/docs/MarkdownHelper.scala

```diff
@@ -0,0 +1,21 @@
+package codacy.codenarc.docs
+
+import better.files.File
+
+import scala.xml.{Elem, XML}
+import sys.process._
+
+object MarkdownHelper {
+
+  def markdownToHtml(markdownFile: File): String = {
+    val html = Seq("pandoc", "-f", "markdown", "-t", "html", markdownFile.path.toString).!!
+    val htmlContent = html.replaceAllLiterally("<br>", "").replaceAllLiterally("<hr>", "")
+    s"<html>$htmlContent</html>"
```

Comment:
Are you sure you need this tag?

Reply:
Yes, because otherwise the returned XML is not well-formed and cannot be parsed